web/client/components/tutorial/style/style.js

```diff
@@ -0,0 +1,149 @@
+/**
+ * Copyright 2017, GeoSolutions Sas.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+const introStyle = {
```

Comment:
We could use css to override the default style but we still need the style to change the tooltips of intro and error. Could we remove the style.js file and include the const introStyle and errorStyle in the Tutorial.jsx component?